### DISCUSSION TITLE
defaultValue text gets truncated with elipsis (...) at the end

### DISCUSSION THREAD

**Discussion by AxRRep** [JedWatson/react-select#5921] [Q&A]
Is there any way to prevent defaultValue getting shortened? It seems like there is no such setting.

![image](https://github.com/JedWatson/react-select/assets/87479163/3495d96a-9f94-4748-b6c7-c8e31d488fa8)

**Comment by manjushsh** [comment]
@AxRRep, can you share codesandbox link with issue?

**Comment by AxRRep** [comment]
Just found out that makeAnimated causes the issue. Removed it and everything works.
